plugins/builder/osbuild.py

```diff
@@ -318,8 +318,7 @@ def handler(self, name, version, distro, image_types, target, arches, opts):
 
         # Architectures
         tag_arches = self.arches_for_config(buildconfig)
-        arches = set(arches)
-        diff = arches - tag_arches
+        diff = set(arches) - tag_arches
```

Comment:
The commit message says to *not* transform arches to a set, but this code is still doing that.

Reply:
It does not transform the `arches` variable. Was obviously not clear enough, sorry.